habitat/plan.ps1

```diff
@@ -68,13 +71,12 @@ function Invoke-Prepare {
 @"%~dp0ruby.exe" "%~dpn0" %*
 "@
         $gem_file | Set-Content "$PWD\\gem.bat"
-        $env:Path += ";c:\\Program Files\\Git\\bin"
+        $env:Path += ";c:\\Program Files\\Git\\bin;"
         $env:HAB_BLDR_CHANNEL = "LTS-2024"
         $env:HAB_STUDIO_SECRET_NODE_OPTIONS = "--dns-result-order=ipv4first"
         $env:HAB_STUDIO_SECRET_HAB_BLDR_CHANNEL = "LTS-2024"
         $env:HAB_STUDIO_SECRET_HAB_FALLBACK_CHANNEL= "LTS-2024"
```

Comment:
are these environment variables needed now that we are setting it above. Specifically `$env:HAB_BLDR_CHANNEL` is getting set in the beginning of this file.